mpu6050/mpu6050.go

```diff
@@ -1,95 +1,251 @@
-// Package mpu6050 provides a driver for the MPU6050 accelerometer and gyroscope
-// made by InvenSense.
-//
-// Datasheets:
-// https://store.invensense.com/datasheets/invensense/MPU-6050_DataSheet_V3%204.pdf
-// https://www.invensense.com/wp-content/uploads/2015/02/MPU-6000-Register-Map1.pdf
-package mpu6050 // import "tinygo.org/x/drivers/mpu6050"
+package mpu6050
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"tinygo.org/x/drivers"
+)
 
-import "tinygo.org/x/drivers"
+const DefaultAddress = 0x68
 
-// Device wraps an I2C connection to a MPU6050 device.
-type Device struct {
-	bus     drivers.I2C
-	Address uint16
+type IMUConfig struct {
+	// Use ACCEL_RANGE_2 through ACCEL_RANGE_16.
+	AccRange byte
```

Comment:
We might want to generalize this, as most accelerometer sensors have exactly this sensor range (2g, 4g, 8g, 16g). It would be nice to be able to use common types for this.

Not very important for this PR, but would perhaps be nice to fix in the future.